lib/hooks/useEvent.lua

```diff
@@ -92,7 +94,18 @@ local function useEvent(instance, event): () -> (number, ...any)
 		storage.queue = queue
 		storage.event = event
 
-		local connection = event:Connect(function(...)
+		-- Get the 'Connect' method through duck typing, since sometimes developers
```

Comment:
The `Disconnect` method hasn't been changed by this PR. I assume that many custom class implementations also return a connection with a key other than `Disconnect`!